__tests__/screens/CredentialOffer.test.tsx

```diff
@@ -1,126 +1,129 @@
-import { CredentialRecord, CredentialState } from '@aries-framework/core'
-import { useCredentialById } from '@aries-framework/react-hooks'
+import { CredentialRecord, ConnectionRecord, CredentialState } from '@aries-framework/core'
+import { useAgent, useCredentialById, useConnectionById } from '@aries-framework/react-hooks'
 import { useNavigation } from '@react-navigation/core'
-import { cleanup, fireEvent, render } from '@testing-library/react-native'
+import { fireEvent, render, waitFor } from '@testing-library/react-native'
+import fs from 'fs'
+import path from 'path'
 import React from 'react'
+import { Alert } from 'react-native'
 
 import CredentialOffer from '../../App/screens/CredentialOffer'
 
-import Button from 'components/buttons/Button'
-import NotificationModal from 'components/modals/NotificationModal'
-
 jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper')
 
-describe('displays a credential offer screen', () => {
-  const testCredentialRecords: CredentialRecord[] = [
-    new CredentialRecord({
-      threadId: '1',
-      state: CredentialState.OfferReceived,
-    }),
-  ]
-
-  afterEach(() => {
-    cleanup()
-  })
+const props = { params: { credentialId: '123' } }
+
+const connectionPath = path.join(__dirname, '../fixtures/faber-connection.json')
+const connection = JSON.parse(fs.readFileSync(connectionPath, 'utf8'))
 
-  beforeEach(() => {
-    jest.clearAllMocks()
+const credentialPath = path.join(__dirname, '../fixtures/degree-credential.json')
+const credential = JSON.parse(fs.readFileSync(credentialPath, 'utf8'))
 
-    testCredentialRecords[0].state = CredentialState.OfferReceived
+const connectionRecord = new ConnectionRecord(connection)
+const credentialRecord = new CredentialRecord(credential)
+// TODO:(jl) Make a fn to revive JSON dates properly and pass to `parse`
+credentialRecord.createdAt = new Date(credentialRecord.createdAt)
```

Comment:
Can use the luxon package used in the app for parsing dates

Reply:
I don't know if it will help here. Standard `Date` can parse ISO just fine. What's needed is a fn that can be passed to JSON.parse() to de-serialize date strings on the fly like `JSON.parse(data, fnToDetectAndDeserializeDates)`.